src/common/externalIntegrations.ts

```diff
@@ -0,0 +1,13 @@
+export interface Integration {
+    name: string;
+    url: string;
+    port: number;
+}
+
+export const externalIntegrations: Integration[] = [
+    {
+        name: 'AUTOMATIC1111/stable-diffusion-webui',
+        url: process.env.STABLE_DIFFUSION_HOST ?? '127.0.0.1',
+        port: Number(process.env.STABLE_DIFFUSION_PORT ?? 7860),
```

Comment:
You are assuming that the env variables for the electron frontend process and the python backend process are the same.

Reply:
I am, because someone typically would start both at the same time with one executable. Also, it's not really assuming anything when we get to be the deciders of what is correct ;)